packages/zipkin/src/console-recorder.js

```diff
@@ -1,19 +1,37 @@
+/**
+ * defaultTags property name
+ * @type {symbol}
+ */
+const defaultTagsSymbol = Symbol('defaultTags');
+
+/**
+ * @class ConsoleRecorder
+ */
 class ConsoleRecorder {
    /* eslint-disable no-console */
   constructor(logger = console.log) {
     this.logger = logger;
   }
+
   record(rec) {
-    const id = rec.traceId;
+    const {spanId, parentId, traceId} = rec.traceId;
+    const defaultTags = this[defaultTagsSymbol] || {};
+    const tags = JSON.parse(defaultTags);
+
     this.logger(
-      `Record at (spanId=${id.spanId}, parentId=${id.parentId},` +
-      ` traceId=${id.traceId}): ${rec.annotation.toString()}`
+      `Record at (spanId=${spanId}, parentId=${parentId}, ` +
+      `traceId=${traceId}, defaultTags=${tags}): ${rec.annotation.toString()}`
```

Comment:
I think this will dramatically increase logging volume, depending on default tags. I'd advise against this.

Reply:
what is the real world use for `ConsoleRecorder`?